ml/trash-recognition/annotations/label_map.pbtxt

```diff
@@ -14,6 +14,6 @@ item {
 }
 
 item {
-    id: 3
+    id: 4
```

Comment:
For anyone who's curious, I screwed up the training of the model by putting a 3 here instead of a 4. This made it so that the model that we're currently using will label cans and tin cans as "food", since they had the same id.